frontend/src/components/footer.tsx

```diff
@@ -23,6 +26,43 @@ const Footer = () => {
                 maxW="1400px"
                 mx="auto"
             >
+                <GridItem colSpan={[1, 2, 4]} color="white" fontSize="md">
+                     <VStack spacing="1em">
+                         <LinkBox transition=".1s ease-out" _hover={{ transform: 'scale(1.05)' }}>
+                             <LinkOverlay isExternal href="https://forms.gle/cRLRxpMap5AX1xx1A">
+                                 <VStack spacing="1em">
+                                     <HStack spacing="1em">
+                                         <Text >
+                                             Har du opplevd noe som ikke føltes helt greit på lesesalen eller på et echo
+                                             arrangement?{' '}
+                                         </Text>
```

Comment:
Den spacen kan fjernes

```suggestion
                                         <Text >
                                             Har du opplevd noe som ikke føltes helt greit på lesesalen eller på et echo
                                             arrangement?
                                         </Text>
```